protocol/examples/query/greeting/wrong_greeting_invalid_address.json

```diff
@@ -0,0 +1,17 @@
+{
+    "jsonrpc": "2.0",
+    "method": "greeting",
+    "params": {
+        "channel": "a channel",
```

Comment:
Isn't the channel invalid there too ? I don't know what is intended, but since the file is called `invalid_address`, I prefer to mention it.

Reply:
Oh right, thank you!